### DISCUSSION TITLE
Can I share assets (images) as a package.

### DISCUSSION THREAD

**Discussion by SandipNirmal** [vercel/turborepo#2868] [Help]
I want to share all images assets with different apps I’ve in my turborepo.

How can I use packages for sharing all the assets.

**Comment by gustavoOchoa** [comment]
ill tell u what i did.
i had a package of components where i link the image in jsx style like this:


```
<style>
    {
    body{
        background-image: url("back_login.jpg");

```
so i give this package a dependency with another package that has css files and images like this:

```
{
  "name": "frmwk",
  "main": "src/index.ts",
  "types": "src/index.ts",
  "dependencies": {
    "typescript": "^4.9.5",
    "shared-styles": "workspace:*"
  }
}
```


and then in my component i did this:

```
import img from 'shared-styles/imgs/back_login_ext.jpg';

url("'+img['src']+'")
```

my css/image package looks like this:
package.json

```
{
  "name": "shared-styles",
  "version": "1.0.0",
  "description": "global styles",
  "main": "index.ts",
  "author": "gurthang",
  "license": "MIT"
}
```


index.ts
```
import * as React from "react";
```

my package structure looks like this:
shared-styles
     -> fonts
     -> icons
    -> imgs
styles.css
index.ts

it works.... buuuuuuuuuuuuuuut gives me the next ts error:

```
import img from 'shared-styles/imgs/back_login_ext.jpg';
//cannot find module shared-styles/imgs/back_login_ext.jpg or its corresponding types declaration
```

dont know why im still fighting with it...
